### PR TITLE
SecurityConfig 프로파일로 환경에 따라 분리

### LINKED ISSUE #26: 인증서버 테스트환경에서 oauth 로그인대신 기본로그인 사용으로 프로파일링

### PR DESCRIPTION
**Pull request by lcc3108**
test 환경에서는 form로그인
일반 환경에서는 oauth 로그인으로 처리
close #26

### CODE REVIEW

**Comment by blackcowmooo** [comment]
Frontend: http://moomark-pr-29.kubernetes.micalgenus.com/
Gateway: http://moomark-pr-29-gateway.kubernetes.micalgenus.com/
Backend.Auth: http://moomark-pr-29-backend-auth.kubernetes.micalgenus.com/

**Comment by blackcowmooo** [comment]
Frontend: http://moomark-pr-29.kubernetes.micalgenus.com/
Gateway: http://moomark-pr-29-gateway.kubernetes.micalgenus.com/
Backend.Auth: http://moomark-pr-29-backend-auth.kubernetes.micalgenus.com/

**Comment by blackcowmooo** [comment]
Frontend: http://moomark-pr-29.kubernetes.micalgenus.com/
Gateway: http://moomark-pr-29-gateway.kubernetes.micalgenus.com/
Backend.Auth: http://moomark-pr-29-backend-auth.kubernetes.micalgenus.com/

**Comment by lcc3108** [comment]
환경변수 적용법 
./gradlew bootRun -Pprofile=production
./gradlew bootRun -Pprofile=development

**Comment by blackcowmooo** [comment]
Frontend: http://moomark-pr-29.kubernetes.micalgenus.com/
Gateway: http://moomark-pr-29-gateway.kubernetes.micalgenus.com/
Backend.Auth: http://moomark-pr-29-backend-auth.kubernetes.micalgenus.com/

**Comment by blackcowmooo** [comment]
Frontend: http://moomark-pr-29.kubernetes.micalgenus.com/
Gateway: http://moomark-pr-29-gateway.kubernetes.micalgenus.com/
Backend.Auth: http://moomark-pr-29-backend-auth.kubernetes.micalgenus.com/

**Comment by blackcowmooo** [comment]
Frontend: http://moomark-pr-29.kubernetes.micalgenus.com/
Gateway: http://moomark-pr-29-gateway.kubernetes.micalgenus.com/
Backend.Auth: http://moomark-pr-29-backend-auth.kubernetes.micalgenus.com/

**Comment by Apisapple** [comment]
내용확인

File: backend/auth/build.gradle [hunk 1]
```diff
@@ -48,3 +48,12 @@ checkstyleMain {
 		xml.destination = file("${checkstyle.reportsDir}/checkstyle-report.xml") // 리포트 파일의 위치 및 파일명 지정
 	}
 }
+
+ext.profile = (!project.hasProperty('profile') || !profile) ? 'development' : profile
+sourceSets {
+    main {
```
Comment (micalgenus) [inline backend/auth/build.gradle:54]: indent 2

**Comment by blackcowmooo** [comment]
Frontend: http://moomark-pr-29.kubernetes.micalgenus.com/
Gateway: http://moomark-pr-29-gateway.kubernetes.micalgenus.com/
Backend.Auth: http://moomark-pr-29-backend-auth.kubernetes.micalgenus.com/

File: backend/auth/build.gradle (hunk 1 above)
Comment (lcc3108) [reply backend/auth/build.gradle]: indent 2로 통일

**Pull request merged**